src/dat/view.cljs

```diff
@@ -266,14 +266,14 @@
 (representation/register-representation
   ::pull-summary-string
   (fn [_ _ pull-data]
-    ;[:span
+    [:span
```

Comment:
::pull-summary-view is what is throwing the error when it calls ::pull-summary-string